examples/multibuilder/src/main.rs

```diff
@@ -1,7 +1,7 @@
 use spirv_builder::SpirvBuilder;
 
 fn main() {
-    let result = SpirvBuilder::new("../shaders/sky-shader")
+    let result = SpirvBuilder::new("../shaders/sky-shader", "spirv-unknown-spv1.3")
```

Comment:
Any chance we can turn this into an enum?

Reply:
Well the environment is already an enum, but I was deliberately keeping this stringly typed as providing the whole target triple is more flexible, as it allows you to put any target here. So if there was an `spirv-embark-spv1.3` target for example it would be valid.